app/BUILD.bazel

```diff
@@ -63,6 +64,8 @@ genrule(
         "//app/components/radio:radio.css",
         "//app/components/checkbox:checkbox.css",
         "//app/components/spinner:spinner.css",
+    ] + [
```

Comment:
Resolved offline, the issue is scoping, but it feels a bit error prone to ensure each V1 style is properly scoped, since it's not as simple as prepending a `.terminal1` prefix to all rules. Will keep as-is